covid-mapper/features/graphs/TimeLineGraph.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import { Dimensions, View, Text } from "react-native";
 import { LineChart } from "react-native-chart-kit";
+import numSeparator from "../../utils/numSeparator";
 
-const chartWidth = Dimensions.get("window").width - 40;
+const chartWidth = Dimensions.get("window").width - 30;
 const chartHeight = Dimensions.get("window").width - 20;
 
```

Comment:
consider setting Dimensions.get("window").width to a variable? Not sure if that would work though.